dev-python/dbus-python/dbus-python-1.2.16-r1.ebuild

```diff
@@ -28,7 +28,10 @@ RDEPEND="${PYTHON_DEPS}
 DEPEND="${RDEPEND}"
 BDEPEND="
 	virtual/pkgconfig
-	doc? ( $(python_gen_any_dep 'dev-python/sphinx[${PYTHON_USEDEP}]') )
+	doc? ( $(python_gen_any_dep '
```

Comment:
You need to update `python_check_deps` too.

Also, I think this whole thing can fall apart if `SPHINX_IMPL` is not in enabled implementations but let's ignore that for now.